.github/workflows/release.yml

```diff
@@ -1,15 +1,25 @@
 name: Auto-release on PR merge
 
 on:
-  # ATM, this is the closest trigger to a PR merging
-  push:
+  # This action should be run in workflows triggered by `pull_request_target`
+  # (not by regular `pull_request`!)
+  pull_request_target:
```

Comment:
Did you copy this from `datalad`?  That uses `release-action`, not `auto`.

Reply:
The point being, I don't think `pull_release_target` is accurate when using `auto`.

Reply:
rright... so it will not have appropriate state when triggered upon merge -- we still have `github.event.pull_request.merged == true` check below so wouldn't it work as desired (run only upon merges etc)?

Reply:
I think you should just undo all changes to everything in the workflow above `steps`, aside from adding in `workflow_dispatch`.

Reply:
> but then it would run the action on every commit/merge into `master`

And `auto` itself takes care of only performing a release if one needs to be made.  This is how [auto's docs recommend doing it](https://intuit.github.io/auto/docs/build-platforms/github-actions), and it's how we do it in every repository where we use `auto`.

> If you think it would not work - I am interested to hear why.

I tried to look into the exact meaning of `pull_request_target` to explain why it might not work, but I just ended up more confused.  Do whatever you want.

Reply:
thanks for pointing to auto's docs. Interesting.  I will try on this PR, if doesn't work -- will redo the "standard way".